web-admin/src/features/billing/plans/selectors.ts

```diff
@@ -53,3 +59,59 @@ export function getNextBillingCycleDate(curEndDateRaw: string): string {
   const nextStartDate = curEndDate.plus({ day: 1 });
   return nextStartDate.toLocaleString(DateTime.DATE_MED);
 }
+
+export function getOrganizationUsageMetrics(
+  organization: string,
+): CreateQueryResult<UsageMetricsResponse> {
+  return derived(
+    [
+      createAdminServiceGetBillingProjectCredentials({
+        organization,
+      }),
+    ],
+    ([credsResp], set) => {
+      if (!credsResp.data) return;
+      return getUsageMetrics(
+        credsResp.data.runtimeHost,
+        credsResp.data.instanceId,
+        credsResp.data.accessToken,
+      ).subscribe(set);
+    },
+  );
+}
+
+export type UsageMetricsResponse = {
+  project_name: string;
+  size: number;
```

Comment:
Can we be more specific than "size"? What's the unit? Size of what exactly?

Reply:
This is in the API response. Lets just add a comment here.